content/docs/replication/deployment/installation.md

```diff
@@ -75,8 +75,9 @@ The following CSI drivers support replication:
 1. CSI driver for PowerMax
 2. CSI driver for PowerStore
 3. CSI driver for PowerScale
+4. CSI driver for PowerFlex
 
-Please follow the steps outlined in [PowerMax](../powermax), [PowerStore](../powerstore) or [PowerScale](../powerscale) pages during the driver installation.
+Please follow the steps outlined in [PowerMax](../powermax), [PowerStore](../powerstore), [PowerScale](../powerscale), or [PowerFlex](../powerflex) pages during the driver installation.
```

Comment:
Similar addition is needed at https://dell.github.io/csm-docs/docs/replication/deployment/storageclasses/#driver-specific-parameters